tidb-lightning/tidb-lightning-faq.md

```diff
@@ -348,3 +348,19 @@ This error occurs usually because the CSV data file does not contain a header (t
 [mydumper.csv]
 header = false
 ```
+
+## How to get goroutine information when TiDB Lightning is running
+
+1. If the configuration file of TiDB Lightning has specified [`status-port`](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-configuration), skip this step. Otherwise, you need to send USR1 signal to TiDB Lightning to enable `status-port`.
```

Comment:
```suggestion
1. If [`status-port`](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-configuration) has been specified in the configuration file of TiDB Lightning, skip this step. Otherwise, you need to send the USR1 signal to TiDB Lightning to enable `status-port`.
```